README.md

```diff
@@ -0,0 +1,10 @@
+TODO: finish README
+
+Online [here](http://microsoft.github.io/TypeSearch).
+
+To update:
+* `gulp build`
+* Move `public` out
+* `git checkout gh-pages`
+* Remove old contents and move contents of `public` in
+* `git checkout master`
```

Comment:
I think you've left our a commit and push command from this description.
Also: Have you considered just writing a gulp task which does this update sequence - `gulp publish`?